WordPress/Classes/Utility/WPWebViewController.m

```diff
@@ -9,19 +9,43 @@
 #import "WPCookie.h"
 #import "Constants.h"
 #import "WPError.h"
+#import "UIAlertView+Blocks.h"
+#import "WordPress-Swift.h"
 
-@class WPReaderDetailViewController;
+
+
+#pragma mark - Constants
+
+static NSInteger const WPWebViewErrorAjaxCancelled          = -999;
+static NSInteger const WPWebViewErrorFrameLoadInterrupted   = 102;
+
+static CGFloat const WPWebViewProgressInitial               = 0.1f;
+static CGFloat const WPWebViewProgressFinal                 = 1.0f;
+
+static CGFloat const WPWebViewAnimationShortDuration        = 0.1f;
+static CGFloat const WPWebViewAnimationLongDuration         = 0.4f;
+static CGFloat const WPWebViewAnimationAlphaVisible         = 1.0f;
+static CGFloat const WPWebViewAnimationAlphaHidden          = 0.0f;
```

Comment:
We could probably leave off the `f` from the const values and let the compiler pick the correct type.